compat/maven-compat/src/main/java/org/apache/maven/repository/legacy/DefaultUpdateCheckManager.java

```diff
@@ -50,8 +50,6 @@ public class DefaultUpdateCheckManager extends AbstractLogEnabled implements Upd
 
     private static final String ERROR_KEY_SUFFIX = ".error";
 
-    public DefaultUpdateCheckManager() {}
```

Comment:
don't change the API

Reply:
Another reason for keeping this constructor is that this class is annotated with `@Named` and `@Singleton`. So there is chances that this constructor is invoked by reflection, even if the IDE reports no usage of this constructor.

Reply:
yes, we apply use (test) or loose.

Reply:
extract thin into dedicated pr just wanted to give attention.